viewer/packages/tools/src/AxisView/AxisViewTool.ts

```diff
@@ -6,6 +6,7 @@ import * as THREE from 'three';
 import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Why changes to AxisViewTool?

Reply:
Converted to be a normal custom object, no need for a UI object backdoor.